sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -103,6 +103,64 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("Reuse the parallelism of CoalescedShuffleReaderExec as the tasks" +
```

Comment:
we can shorten the test name: `Reuse the parallelism of CoalescedShuffleReaderExec in LocalShuffleReaderExec`